src/style.css

```
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-size: 16px;
+  padding-top: 12px;
+}
+
+.form {
+  position: relative;
+}
+
+.form_search {
+  width: 100%;
+  font-family: "Avenir-Heavy", Arial, sans-serif ;
+  font-size: inherit;
+  font-weight: inherit;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
```

Зроби відступи між різними групами CSS правил, тоді твій код буде простіше читати і підтримувати
```suggestion
  width: 100%;
  
  font-family: "Avenir-Heavy", Arial, sans-serif ;
  font-size: inherit;
  font-weight: inherit;
  
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
  box-sizing: border-box;
```